_posts/2018-07-11-OSBAR-award-2017.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "The Rust project wins an OSBAR award"
+lang: en-GB
+author: Florian Gilcher
+---
+
+Last winter, the Rust project has won the bronze award of the Open Source Business Alliance Awards (OSBAR), an award given by the [Open Source Business Alliance](http://osb-alliance.de/) to open source projects that bring benefit both to companies and public entities.
+
+The award was given in appreciation of our successful community building and our push to provide safer programming in areas were they were previously not feasible.
+
+We've been applying for this award multiple times. Writing down an application and having to argue how Rust brings a notable benefit has been a fun experience.
+
+This year, Florian Gilcher will be part of the jury for [the award](https://award.osb-alliance.de/) and would like to encourage fitting Rust projects to apply. Projects submitted should have value for business or for public administration, but should also be proper open source projects.
+
+In the interest of transparency, the full (unedited) application text follows at the end of this post.
```

Comment:
maybe explicitly mention that it's last year's Rust application (as just before you were speaking about this year)

Reply:
Or even better, maybe move this sentence up to line 12?